.gitlab-ci.yml

```diff
@@ -2,6 +2,13 @@ image: docker:git
 
 variables:
   DOCKER_REPOSITORY: mendersoftware/mender-convert
+  S3_BUCKET_NAME: mender-convert-images
+  MENDER_ARTIFACT_VERSION: master
+  MENDER_CLIENT_VERSION: master
```

Comment:
Once we release (say we branch out for 2.0.x), then this line will be modified to point to stable Mender client (2.2.0, most likely).

Or do you have other idea in mind?

Reply:
Yes, but we need `mender setup`, don't we? I recall you mentioning something about needing the master branch until 2.3, but I can't recall the exact details.

Reply:
Discussed in person.

We are both referring to the same thing: Mender 2.3 which will include client 2.2.0